openvino/itt/include/openvino/itt.hpp

```diff
@@ -0,0 +1,173 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+/**
+ * @brief Defines API to trace using Intel ITT.
+ * @file itt.hpp
+ */
+
+#pragma once
+#include "function_name.hpp"
+#include "macro_overload.hpp"
```

Comment:
I prefer `openvino/function_name.hpp`

Reply:
It is the same. However, I'll change it.